src/material/core/m2/_palette.scss

```diff
@@ -688,6 +688,7 @@ $light-theme-background-palette: (
   unselected-chip: map.get($grey-palette, 300),
   disabled-list-option: map.get($grey-palette, 200),
   tooltip: map.get($grey-palette, 700),
+  stepper-icon: $dark-secondary-text,
```

Comment:
I think at this stage we probably don't want to add new entires to this map, but it should be ok to redefine the tokens in terms of these existing properties

Reply:
@mmalerba For the background color I can revert the changes, but for the foreground colors the problem is that I could not find any value in the existing palettes which produces white for light themes and black for dark themes. How exactly can I define these values, without extending the palettes?

Reply:
@mmalerba I eventually figured it out, please review again.